serverless.yml

```diff
@@ -397,6 +402,29 @@ functions:
               - name: request.querystring.id
               - name: request.header.origin
 
+  node_api_get_tokens:
+    handler: handlers/token_api.get_tokens
+    maximumRetryAttempts: 0
+    package:
+      patterns:
+        - 'handlers/token_api.py'
+    layers:
+      - { Ref: PythonRequirementsLambdaLayer }
+    events:
+      - http:
+          path: token_api/tokens
+          method: get
+          cors: true
+          caching:
+            enabled: true
+            ttlInSeconds: 300
```

Comment:
Just to confirm, using the same `cacheKeyParameters` below we should always get the same results?

Reply:
Yes, we want to reduce a little bit the load on ElasticSeach with this cache enabled. For the same parameters, the same results must be returned. As a downside, the user might get outdated data for 5 minutes.